aleth/main.cpp

```diff
@@ -756,13 +752,11 @@ int main(int argc, char** argv)
              << "WALLET USAGE:\n";
         AccountManager::streamAccountHelp(cout);
         AccountManager::streamWalletHelp(cout);
-        cout << clientDefaultMode << clientTransacting << clientMining << clientNetworking;
-        MinerCLI::streamHelp(cout);
+        cout << clientDefaultMode << clientTransacting << clientMining << minerOptions << clientNetworking;
```

Comment:
Could you move `clientNetworking` before `clientMining` here, I think networking options better should be more visible than after everything about mining.